### PR TITLE
feat: Add emergency pie delivery

### PR DESCRIPTION
**Pull request by DisposableCrewmember42**
Added emergency pie delivery crate, containing 4 banana cream pies and setting Logistics back by ~~666 Spesos~~. The bounty integration tests don't want us to have nice things. The crate now costs 3333 Spesos and contains 13 banana cream pies.

## Why / Balance
Requested [in Discord](https://discord.com/channels/968983104247185448/1307145855199084644/1307150175780733009).

## Technical details
Added FoodEmergencyPie and CrateFoodEmergencyPie prototypes along with necessary i18n

## Media
None

## Requirements

- [x] I have tested all added content and changes.
- [x] I have added media to this PR or it does not require an ingame showcase.

## Breaking changes

**Changelog**

:cl: DisposableCrewmember42
- add: Hungry? Stuff your face with cream from an emergency pie delivery for a change!

### CODE REVIEW

File: Resources/Prototypes/DeltaV/Catalog/Fills/Crates/food.yml [hunk 1]
```diff
@@ -29,3 +29,12 @@
           amount: 1
         - id: FoodBoxDonkpocketDink
           prob: 0.1
+
+- type: entity
```
Comment (deltanedas) [inline Resources/Prototypes/DeltaV/Catalog/Fills/Crates/food.yml:33]: this should have the locale here for english, ent- is for translation

**Review by deltanedas — CHANGES_REQUESTED** [review]

File: Resources/Prototypes/DeltaV/Catalog/Fills/Crates/food.yml [hunk 2]
```diff
@@ -29,3 +29,14 @@
...
+  id: CrateFoodEmergencyPie
+  parent: CratePlastic
+  name: emergency pie delivery
+  description: '"Then let them eat pie."'
+  components:
+    - type: StorageFill
```
Comment (deltanedas) [inline Resources/Prototypes/DeltaV/Catalog/Fills/Crates/food.yml]: unindent a level

File: Resources/Prototypes/DeltaV/Catalog/Fills/Crates/food.yml [hunk 3]
```diff
@@ -29,3 +29,14 @@
...
+      contents:
+        - id: FoodPieBananaCream
```
Comment (deltanedas) [inline Resources/Prototypes/DeltaV/Catalog/Fills/Crates/food.yml]: unindent a level

**Review by deltanedas — CHANGES_REQUESTED** [review]

**Comment by DisposableCrewmember42** [comment]
*screams*

**Review by deltanedas — APPROVED** [review]

**Pull request merged**